### PR TITLE
Reimplement users spreadsheet with caxlsx gem

### PR DESCRIPTION
**Pull request by tonyaraujop**
Closes: #258

### CODE REVIEW

File: spec/spreadsheets/users_spreadsheet_spec.rb [hunk 1]
```diff
@@ -47,23 +47,21 @@
...
+    subject(:spreadsheet) { users_spreadsheet.generate_xlsx }
 
     it 'returns spreadsheet object with header' do
-      expect(spreadsheet.row(0)).to containing_exactly(*header_attributes)
+      expect(spreadsheet.rows.first.cells.map(&:value)).to include(*header_attributes)
     end
 
     it 'returns spreadsheet object with body' do
-      expect(spreadsheet.row(1)).to containing_exactly(*body_attributes)
+      expect(spreadsheet.rows.last.cells.map(&:value)).to include(*body_attributes)
```
Comment (adbatista) [inline spec/spreadsheets/users_spreadsheet_spec.rb:64]: why the change to include?

Comment (tonyaraujop) [reply spec/spreadsheets/users_spreadsheet_spec.rb:64]: Sorry, I was getting an error and forgot to change it back. I'll revert to `containing_exactly`
